mods/essb/abilities.js

```diff
@@ -350,6 +350,18 @@ exports.BattleAbilities = {
 			}
 		},
 	},
+	//flufi
+	"thewaggling": {
+		id: "thewaggling",
+		name: "The Waggling",
+		onStart: function (pokemon) {
+			this.useMove("Metronome", pokemon);
```

Comment:
I think this may be too strong because well Metronome could allow various things to occur